addons/sys_rack/stringtable.xml

```diff
@@ -144,6 +146,7 @@
             <Chinesesimp>仪表板下降</Chinesesimp>
             <Spanish>P.Inf</Spanish>
             <French>T.Inf</French>
+            <German>Unterster Steckplatz</German>
```

Comment:
have you checked that it's not too long (decluttering of the Menu is vital)

Reply:
This one should be maximum 5 characters or a warning will be logged.